docs/Zendesk-batchsource.md

```diff
@@ -48,19 +47,19 @@ the specified date. The date must be provided in the date format:
 |                                  | YYYY-MM-DDThh:mm:ss-hh:mm | 1999-01-01T23:01:01-08:00 |
 |                                  | YYYY-MM-DDThh:mm:ssZ      | 1999-01-01T23:01:01Z      |
 
-Specifying this along with `Start Date` allows reading data modified within a specific time window.
+If you enter an End Date and Start Date, the data modified within a specific time window.
```

Comment:
I think we are missing "is: after "data".  "...the data is modified within a specific time window."

Reply:
added